### DISCUSSION TITLE
Show reference from Releases

### DISCUSSION THREAD

**Discussion by landsman** [community/community#43211] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I really like how it shows that someone has mentioned the Pull Request in an Issue or other Pull Request.

I'd love it if the same thing worked with GitHub Releases.

We use Releases for product documentation, individual releases to production. 
Sometimes it's handy to see at the PR level that it was included in a Release.

Is this possible for you, does it make sense?

**Comment by LuisContrerasGlz** [comment]
it is possible to see which Pull Requests were included in a GitHub Release, To see this information, you can go to the "Commits" tab of the Release page. This will show a list of all the commits that were included in the Release, along with the Pull Requests that those commits were associated with. Also you can go to the "Files changed" tab of the Release page, which will show a list of all the files that were changed as part of the Release. If you click on a file, you can see the commit that introduced the change, and from there you can see the Pull Request that the commit was associated with.

**Reply by landsman** [reply]
I manually writing Release Notes and adding there bullet points with Pull Request in this syntax:

- some change (#123456)
- some change 2 (#1234567)

so thanks to this syntax I would expect to see that I mentioned this PR in the Release
